tests/ctst/steps/azureArchive.ts

```diff
@@ -468,7 +477,8 @@ Then('object {string} should expire in {int} days', async function (this: Zenko,
     const exiryDate = new Date(expireResDate[1]).getTime();
     const lastModified = new Date(head.LastModified).getTime();
     const diff = (exiryDate - lastModified) / 1000 / 86400;
-    assert(diff >= days && diff < days + 0.1);
+    days = days / (this.parameters.TimeProgressionFactor ? this.parameters.TimeProgressionFactor : 1);
```

Comment:
can be simplified like the bash code, to protect against other values

```suggestion
        (this.parameters.TimeProgressionFactor > 1 ? this.parameters.TimeProgressionFactor : 1);
```